tests/src/test/scala/cats/mtl/tests/Syntax.scala

```diff
@@ -82,6 +97,12 @@ final class Syntax extends BaseSuite {
       val tupled: WriterT[Option, String, Unit] = ("ha", ()).tuple[WriterTC[Option, String]#l]
       val toldC = Tell.tell[WriterTC[Option, String]#l, String]("ha")
       val toldFC = Tell.tellF[WriterTC[Option, String]#l]("ha")
+
+      def bar[F[_]](implicit F: Tell[F, Bar]) =
+        F.tell(Bar(404))
+
+      def foo[F[_]](implicit F: Tell[F, Foo]): F[Unit] =
+        bar[F]
```

Comment:
I think in general we really want to check the syntax resolution more than the raw subtyping. I'm pretty sure most of it will work, but I'm worried about that more than I am relatively simple subsumption cases like this one.